news/2019-11-14-beatmap-pack-management-update-1.md

```diff
@@ -0,0 +1,63 @@
+
+### Best of 2010–2011 pack
+
+*created by [Dada](https://osu.ppy.sh/users/9119507), [\_p0ke\_](https://osu.ppy.sh/users/5434711), [Halfslashed](https://osu.ppy.sh/users/4598899), [Uberzolik](https://osu.ppy.sh/users/1314547), and [Cynplytholowazy](https://osu.ppy.sh/users/3901754)*  
+*written by [\_p0ke\_](https://osu.ppy.sh/users/5434711)*  
+[Download](https://osu.ppy.sh/beatmaps/packs/1733)  
+
+Ever wanted to see what popular maps looked like in earlier eras of the game? Well now you can! Forged together by members of the Beatmap Pack Management team, you can check out some of the best maps that the game had to offer during 2010 and 2011 in a single pack. While there is a lot of great maps, the team didn't have enough room to include everything, so hopefully you can find something on this list that you will enjoy. Play through them all and experience a blast from the past!
+
+### osu!dan project
+
+*written by [celerih](https://osu.ppy.sh/users/4696296)*  
+
+The osu!dan project is a brand new community created project that seeks to provide a "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.
+
+Each of the dans will contain 20 levels of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, like reaching a certain accuracy threshold or attaining a full combo. For now, the release of dan levels is purely for fun, a way to practice, or test yourself, but a more official implementation is planned for the future. 😉
+
+The first of the dans is available to download now on the Beatmap Packs listing. The skill tested in this pack is aim and will surely push your circle clicking to the limits. The requirements for this dan are quite simple: you need to achieve full combo on 10 out of 11 maps in the dan to progress to the next one. Once again, this for now is just for your own practice, or to compare yourself with your friends - there is no implementation for keeping track of levels as for now.
+
+If you wish to help with playtesting future packs and want to stay up to date, you can join the official [osu!dan Discord server](https://discord.gg/b7cVmGd) now!
+
+### October Loved packs
+
+*written by [Stefan](https://osu.ppy.sh/users/626907)*
+
+As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since mid October. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```

Comment:
```suggestion
The time came once again to bundle a few beatmaps, with some new Loved beatmap packs being made available in mid October. That said, I want to take this chance to explain why Loved beatmaps do not get bundled into packs right away. Since there is always a small chance that a beatmap can become unloved, I decided to give maps at least a minimum of 10 months to stay in Loved, and then afterwards bundle the Loved beatmaps into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives!
```

Reply:
mostly putting it in past tense since this news post got delayed so much that saying "it is time" isn't accurate anymore (unless you decide to additionally publish november and december loved packs now)